src/network/networkd-json.c

```diff
@@ -1161,6 +1162,143 @@ static int dns_misc_build_json(Link *link, JsonVariant **ret) {
         return r;
 }
 
+static int dhcp_server_offered_leases_build_json(Link *link, JsonVariant **ret) {
+        JsonVariant **elements = NULL;
+        size_t n = 0;
+        int r;
+        DHCPLease *lease;
+
+        assert(link);
+        assert(ret);
+
+        if (!link->dhcp_server) {
+                *ret = NULL;
+                return 0;
+        }
+
+        HASHMAP_FOREACH(lease, link->dhcp_server->bound_leases_by_client_id) {
+                _cleanup_free_ char *id = NULL;
+                if (!GREEDY_REALLOC(elements, n + 1)) {
```

Comment:
Please insert an empty line after the variable declaration.

Reply:
Obsolete.